trojsten/web
# Fix pip version in Dockerfile

- Pip 20.1 breaks stupid packages which import non-documented pip internals in their `setup.py`.
- This ensures we are using Pip 20.0.2 which is the latest version that seems to be working.

**How to test:**
- First run `docker pull python:3.8-alpine3.11` to ensure that you have the latest python image which includes the latest pip version
- Try to build master with it, it should fail on `python-akismet`.
- Try to build this branch with it, it should succeed.
